lib/models/WebsiteConfiguration.ts

```diff
@@ -77,10 +77,10 @@ export type RedirectAllRequestsTo = {
 };
 
 export type WebsiteConfigurationParams = {
-    indexDocument: string;
-    errorDocument: string;
-    redirectAllRequestsTo: RedirectAllRequestsTo;
-    routingRules: RoutingRule[] | any[],
+    indexDocument?: string;
+    errorDocument?: string;
+    redirectAllRequestsTo?: RedirectAllRequestsTo;
+    routingRules?: RoutingRule[] | any[],
```

Comment:
does not seem really typed....

Reply:
I updated it, due to a wrong type being used in BucketInfo (class type instead of the "Params" version)

Reply:
Actually, the whole website implementation is a mess: js code uses private members of the website class and we eneded up storing that in mongodb, so we need to keep this behavior... Working on it

Reply:
Done...